### ISSUE TITLE
This version of Structure app is too old for CaputreReceiverExample

### ISSUE THREAD

**Issue by mehranhussain** [occipital/uplink#2]
I'm getting this error message while connecting to uplink on Structure Sensor app on my ipad.
Error dictates "You should update the structure app to its latest version" but currently my structure app is up-to-dated to its 2.0 firmware version.

**Comment by gzhang8** [comment]
Yes, I run into the same problem. Did you solved it? @mehranhussain

**Comment by mehranhussain** [comment]
No, I think its happening because of the function call mismatch between Structure app and uplink server listener.
